### PR TITLE
non-linked text no longer looks like a link

### PR DESCRIPTION
**Pull request by cliffordp**
See central ticket for screenshots

https://central.tri.be/issues/72475#note-11

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$venue_address_tag' | [email.php:391](https://github.com/moderntribe/event-tickets/blob/444259459eb03aefbe68c72f9cd38fe6c16e089a/src/views/tickets/email.php#L391)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$venue_map_url' | [email.php:391](https://github.com/moderntribe/event-tickets/blob/444259459eb03aefbe68c72f9cd38fe6c16e089a/src/views/tickets/email.php#L391)

> via jenkins/codesniffer-phpcs

File: src/views/tickets/email.php [hunk 1]
```diff
@@ -388,7 +388,7 @@
 																<tr>
 																	<td class="ticket-venue-child" valign="top" align="left" width="130" style="padding: 0 10px 0 0 !important; width:130px; margin:0 !important;">
 																		<span style="color:#0a0a0e !important; font-family: 'Helvetica Neue', Helvetica, sans-serif; font-size:13px; display:block; margin-bottom:5px;"><?php echo $venue_name; ?></span>
-																		<<?php echo $venue_address_tag; ?> style="color:#006caa !important; display:block; margin:0; font-family: 'Helvetica Neue', Helvetica, sans-serif; font-size:13px; text-decoration:underline;" <?php if ( 'a' === $venue_address_tag ) { printf( 'href="%s"', $venue_map_url ); } ?>>
+																		<<?php echo $venue_address_tag; ?> style="<?php if ( ! empty( $venue_map_url ) ) { echo 'color:#006caa !important; text-decoration:underline; '; } ?>display:block; margin:0; font-family: 'Helvetica Neue', Helvetica, sans-serif; font-size:13px;" <?php if ( 'a' === $venue_address_tag ) { printf( 'href="%s"', $venue_map_url ); } ?>>
```
Comment (barryhughes) [inline src/views/tickets/email.php]: * Won't the link inherit normal link styling in any case, removing the need for this? If not, can we add the necessary rules to the `<style>` element instead? That would make this line far more legible
* This code is already pretty hard to read (I realize you're just adopting the existing style but I'm really reluctant to compound things by making the line even longer) ... if the last suggestion isn't viable how about forming the style attribute and placing it inside a var a little earlier, then echoing that var?

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | Whitespace found at end of line | [email.php:276](https://github.com/moderntribe/event-tickets/blob/bbffd1df8e36d4580b11068ccc89963fb9fe35da/src/views/tickets/email.php#L276)
:x: | Whitespace found at end of line | [email.php:289](https://github.com/moderntribe/event-tickets/blob/bbffd1df8e36d4580b11068ccc89963fb9fe35da/src/views/tickets/email.php#L289)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$venue_address_tag' | [email.php:396](https://github.com/moderntribe/event-tickets/blob/bbffd1df8e36d4580b11068ccc89963fb9fe35da/src/views/tickets/email.php#L396)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$venue_address_style' | [email.php:396](https://github.com/moderntribe/event-tickets/blob/bbffd1df8e36d4580b11068ccc89963fb9fe35da/src/views/tickets/email.php#L396)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$venue_map_url' | [email.php:396](https://github.com/moderntribe/event-tickets/blob/bbffd1df8e36d4580b11068ccc89963fb9fe35da/src/views/tickets/email.php#L396)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$venue_address_tag' | [email.php:396](https://github.com/moderntribe/event-tickets/blob/209f912dc65b55c185d3f5a0878d7a22bc2d3cb7/src/views/tickets/email.php#L396)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$venue_map_url' | [email.php:396](https://github.com/moderntribe/event-tickets/blob/209f912dc65b55c185d3f5a0878d7a22bc2d3cb7/src/views/tickets/email.php#L396)

> via jenkins/codesniffer-phpcs

**Pull request merged**